src/lib.rs

```diff
@@ -314,6 +314,30 @@ impl Ext4 {
 
         inner(self, path.try_into().map_err(|_| Ext4Error::MalformedPath)?)
     }
+
+    /// Check if `path` exists.
+    ///
+    /// Returns `Ok(true)` if `path` exists, or `Ok(false)` if it does
+    /// not exist.
+    ///
+    /// # Errors
+    ///
+    /// An error will be returned if:
+    /// * `path` is not absolute.
```

Comment:
Just noticed that here and in #114 the comment doesn't quite match the tests. I can see an argument for not listing 'malformed path', though in #114 it looks like the malformed test might relate to the 'not exist' comment? Or is there a missing test there?

Just pointing it out in case it's an oversight, rather than intentional.

Reply:
I've been vaguely thinking that for common errors I'd put some docs in the top-level doc for Ext4. Otherwise each of these methods would have to list malformed path, symlink loop, corrupt filesystem, path too long, etc, which would get a bit unwieldy. I should probably add a "this list is not exhaustive" note to each of these methods though, like `std::fs` does: https://doc.rust-lang.org/std/fs/fn.read_dir.html#errors

Reply:
Filed https://github.com/nicholasbishop/ext4-view-rs/issues/119